fetch/h1-parsing/resources/status-code.py

```diff
@@ -1,6 +1,6 @@
 def main(request, response):
-    output = "HTTP/1.1 "
-    output += request.GET.first("input")
-    output += "\n" + "header-parsing: is sad" + "\n"
+    output = b"HTTP/1.1 "
+    output += request.GET.first(b"input")
+    output += b"\n" + b"header-parsing: is sad" + b"\n"
```

Comment:
Why isn't this just one string literal?

Reply:
I'm not sure. I copied it from #5102. (And that's too long ago to remember.) I can flatten it.